raft/server.cc

```diff
@@ -1441,8 +1452,8 @@ future<> server_impl::wait_for_apply(index_t idx, abort_source* as) {
         // This will be signalled when read_idx is applied
         auto it = _awaited_indexes.emplace(idx, awaited_index{{}, {}});
         if (as) {
-            it->second.abort = as->subscribe([this, it] () noexcept {
-                it->second.promise.set_exception(request_aborted());
+            it->second.abort = as->subscribe([this, it]() noexcept {
+                it->second.promise.set_exception(request_aborted(format("Aborted while waiting to apply entry: {}", it->first)));
```

Comment:
This message is less rich in details than the message above.

Reply:
Copied & added applied_idx

Reply:
This one looks unresolved, I don't see `applied_idx` here